src/tests/test_mockwebserver.py

```diff
@@ -23,35 +24,33 @@ def test_get_page(self):
         with self.server:
             response = self.get('page')
             self.failUnless(response.ok)
```

Comment:
`failUnless` is a deprecated alias for `assertTrue` (https://docs.python.org/2/library/unittest.html#deprecated-aliases).

```suggestion
             self.assertTrue(response.ok)
```